### PR TITLE
Tier3-CEPH-83575762

### PR DESCRIPTION
**Pull request by hkadam134**
TC : https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-83575762
Logs : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-SMDSM3/

# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: tests/cephfs/cephfs_bugs/test_create_unlink_ops_concorrently.py [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+                        sudo=True, cmd=f"rm -rf {directory}{filename}"
+                    )
+        log.info(
+            "Check for the Ceph Health to see if there are any crash and Health Errors."
+        )
+        ceph_health = fs_util_v1.get_ceph_health_status(clients[0])
+        log.info(ceph_health)
+        return 0
+    except Exception as e:
+        log.info(e)
```
Comment (AmarnatReddy) [inline tests/cephfs/cephfs_bugs/test_create_unlink_ops_concorrently.py:156]: can you add error instead of info

**Review by AmarnatReddy — COMMENTED** [review]
Small Comment Rest Looks good

File: tests/cephfs/cephfs_bugs/test_create_unlink_ops_concorrently.py (hunk 1 above)
Comment (hkadam134) [reply tests/cephfs/cephfs_bugs/test_create_unlink_ops_concorrently.py]: Fixed

**Review by neha-gangadhar — APPROVED** [review]
lgtm

**Pull request merged**